src/App.tsx

```diff
@@ -681,6 +767,29 @@ const MapDisplay = (
                       (100 / (zoomLevel || 100))
                     )
 
+                    // Road Click
+                    let road = -1
+
+                    if (clickX >= 45 && clickX <= 55) {
+                      if (clickY < 50) {
+                        road = 2
+                      } else {
+                        road = 0
+                      }
+                    } if (clickY >= 45 && clickY <= 55) {
+                      if (clickX < 50) {
+                        road = 1
+                      } else {
+                        road = 3
+                      }
+                    }
+
+                    if (road !== -1) {
```

Comment:
Clicking on a something that is not a road triggers this. You need to check that the clicked tile actually has road on the position that was clicked. Otherwise the code will select every non road point as debug road.